src/display/views/help/components/__tests__/__snapshots__/ContactFormPane.spec.js.snap

```diff
@@ -18,117 +18,34 @@ exports[`rendering snapshots should render valid snapshot 1`] = `
       className="card-body"
       tag="div"
     >
-      <AvForm
-        data-spec="contact-form-container"
-        method="get"
-        model={Object {}}
-        onInvalidSubmit={[Function]}
-        onKeyDown={[Function]}
-        onSubmit={[Function]}
-        onValidSubmit={[Function]}
-        tag={[Function]}
-        validationEvent={
-          Array [
-            "onChange",
-            "onInput",
-          ]
-        }
+      <p>
+        Have a suggestion about the site? Encountered a bug? Want to just say hi or give me a hug? Click th ebutton below to open an issue on GitHub with your message.
```

Comment:
Quick fix here - typo on "click the button"

Reply:
On it rn, thanks for pointing it out!